src/cljs/netrunner/deckbuilder.cljs

```diff
@@ -425,7 +406,9 @@
      [:div {:class (if (:legal cache-refresh) "legal" "invalid") :title (if onesies-details? (:reason cache-refresh)) }
       [:span.tick (if (:legal cache-refresh) "✔" "✘")] "Cache Refresh compliant"]
      [:div {:class (if (:legal onesies) "legal" "invalid") :title (if onesies-details? (:reason onesies))}
-      [:span.tick (if (:legal onesies) "✔" "✘") ] "1.1.1.1 format compliant"]]))
+      [:span.tick (if (:legal onesies) "✔" "✘") ] "1.1.1.1 format compliant"]
+     [:div {:class (if (:legal modded) "legal" "invalid") :title (if onesies-details? (:reason modded))}
```

Comment:
I'd rename `onesies-details?` to something more accurate / descriptive now.

Reply:
Also, there are 3 more or less identical `:div` structures here, just begging to be extracted out.